### PR TITLE
Sync ext/dom properties with stubs - part 1

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/dom/domdocumentfragment.xml [hunk 1]
```diff
@@ -95,6 +113,13 @@
       </row>
      </thead>
      <tbody>
+      <row>
+       <entry>8.0.0</entry>
+       <entry>
+        The <varname>firstElementChild</varname>, <varname>lastElementChild</varname>,
+        and <varname>childElementCount</varname> properties are added now.
```
Comment (Girgias) [inline reference/dom/domdocumentfragment.xml:120]:
```suggestion
        and <varname>childElementCount</varname> properties have now been added.
```
or just
```suggestion
        and <varname>childElementCount</varname> properties have been added.
```

**Review by Girgias — CHANGES_REQUESTED** [review]
Phrasing needs to be updated for the other files

File: reference/dom/domdocumentfragment.xml (hunk 1 above)
Comment (kocsismate) [reply reference/dom/domdocumentfragment.xml]: Fixed! Can you please have another look sometime? :)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**